radioapp/ui/ui_backlight.py

```diff
@@ -0,0 +1,62 @@
+from tkinter import *
+
+
+class UIBackLight():
+	"""Creates UI Backlight Screen."""
+
+	def __init__(self):

```

Comment:
There's no way to escape this menu, but there should be. The back button should be configured.

Reply:
Yes, I know it needs back button, it needs to save it's settings and what not, but the UI MVP needs refactoring, once that's done I can easily add this to the MVP. This is just a UI backlight screen to show how it will look like and how it will function, when it's integrated in MVP, certain things like background color, geometry, mainloop will be removed. I should have addressed this in the description, my bad.

Reply:
This is not complete (and will not be merged or marked as done) until it is integrated with the rest of the codebase (at least the UI).

Reply:
Again, I am not sure what refactoring needs to be done for a back button. If I'm not mistaken there was an example page with a back button.